### ISSUE TITLE
Got it working but few Warnings

### ISSUE THREAD

**Issue by npscode** [alfredfrancis/ai-chatbot-framework#133]
using Tensorflow v1.15.2 

but while training model i get warnings given below. Please suggest, do i need to upgrade tensorflow or do i need to do something else to overcome these warnings.

WARNING:tensorflow:From /var/www/html/chatbot/app/nlu/classifiers/starspace_intent_classifier.py:485: The name tf.placeholder is deprecated. Please use tf.compat.v1.placeholder instead.

WARNING:tensorflow:From /var/www/html/chatbot/app/nlu/classifiers/starspace_intent_classifier.py:492: The name tf.placeholder_with_default is deprecated. Please use tf.compat.v1.placeholder_with_default instead.

WARNING:tensorflow:
The TensorFlow contrib module will not be included in TensorFlow 2.0.
For more information, please see:
  * https://github.com/tensorflow/community/blob/master/rfcs/20180907-contrib-sunset.md
  * https://github.com/tensorflow/addons
  * https://github.com/tensorflow/io (for I/O related ops)
If you depend on functionality not listed there, please file an issue.

WARNING:tensorflow:From /var/www/html/chatbot/app/nlu/classifiers/starspace_intent_classifier.py:272: dense (from tensorflow.python.layers.core) is deprecated and will be removed in a future version.
Instructions for updating:
Use keras.layers.Dense instead.
WARNING:tensorflow:From /var/www/html/chatbot/npf-chatbot/lib/python3.7/site-packages/tensorflow_core/python/layers/core.py:187: Layer.apply (from tensorflow.python.keras.engine.base_layer) is deprecated and will be removed in a future version.
Instructions for updating:
Please use `layer.__call__` method instead.
WARNING:tensorflow:From /var/www/html/chatbot/app/nlu/classifiers/starspace_intent_classifier.py:273: dropout (from tensorflow.python.layers.core) is deprecated and will be removed in a future version.
Instructions for updating:
Use keras.layers.dropout instead.
WARNING:tensorflow:From /var/www/html/chatbot/app/nlu/classifiers/starspace_intent_classifier.py:323: The name tf.losses.get_regularization_loss is deprecated. Please use tf.compat.v1.losses.get_regularization_loss instead.

WARNING:tensorflow:From /var/www/html/chatbot/app/nlu/classifiers/starspace_intent_classifier.py:497: The name tf.train.AdamOptimizer is deprecated. Please use tf.compat.v1.train.AdamOptimizer instead.

WARNING:tensorflow:From /var/www/html/chatbot/npf-chatbot/lib/python3.7/site-packages/tensorflow_core/python/ops/math_grad.py:1424: where (from tensorflow.python.ops.array_ops) is deprecated and will be removed in a future version.
Instructions for updating:
Use tf.where in 2.0, which has the same broadcast rule as np.where
WARNING:tensorflow:From /var/www/html/chatbot/app/nlu/classifiers/starspace_intent_classifier.py:368: The name tf.global_variables_initializer is deprecated. Please use tf.compat.v1.global_variables_initializer instead.
